packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -286,18 +322,27 @@ export class AtomicSearchBox {
       }
     });
 
-    const suggestionElements = fulfilledSuggestions
-      .sort((a, b) => a.position - b.position)
-      .map((suggestion) => suggestion.renderItems())
-      .flat();
+    this.leftSuggestions = fulfilledSuggestions
```

Comment:
nitpick: since it's repetitive for both side, just a small inline function should work.

```
const splitSuggestions = (side: 'left' | 'right') => fulfilledSuggestions
      .filter((suggestion) => suggestion.panel === side)
      .sort(this.sortSuggestions);

const left = splitSuggestions('left');
const right = splitSuggestions('right');
```

Reply:
I thought about it but then we don't default to the left panel when not available.. but I guess I could add that into the function too!